<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>animalDetailView</class>
 <widget class="QDialog" name="animalDetailView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Animal</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>161</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu</string>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <zorder>saveButton</zorder>
   <zorder>editButton</zorder>
   <zorder>addButton</zorder>
   <zorder>backButton</zorder>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Add Animal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="fieldsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>371</width>
      <height>10000000</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>355</width>
       <height>669</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="animalName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>64</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalType">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Dog</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bird</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Small Animal</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>64</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Detail:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="animalDetail">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>261</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sex:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalSex">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>M</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="animalAge">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>64</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Age:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="animalColour">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>261</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>64</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Colour:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>64</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Lifespan:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="animalLifespan">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="animalDifficulty">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Easy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Intermediate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Difficult</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Difficulty of Care:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalAffection">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Little</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moderate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lots</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Affection Needed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>300</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Average Cost per Year:     $</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="animalCost">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="animalTime">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Time Commitment per Day:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>340</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>hours</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalSpace">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>380</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Little Space</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moderate Space</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lots of Space</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Space Needed:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalLoudness">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>420</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>No Noise</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Little Noise</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moderate Noise</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lots of Noise</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Noise Made:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalActiveness">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Little</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moderate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lots</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Activity Required:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="animalIntegration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Integration</string>
      </property>
      <widget class="QCheckBox" name="dogCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Dog</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">animalButtonGroup</string>
       </attribute>
      </widget>
      <widget class="QCheckBox" name="childCheckBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Children</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">animalButtonGroup</string>
       </attribute>
      </widget>
      <widget class="QCheckBox" name="catCheckBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cat</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">animalButtonGroup</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obedience Level:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalObedience">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>590</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Untrained</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Somewhat Trained</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Trained</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Animal Shedding:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalShedding">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>630</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Little</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Moderate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lots</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="animalSize">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>S</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>L</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="animalButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
